### ISSUE TITLE
Add `blurb` to `cpython` PyPI organisation

### ISSUE THREAD

**Issue by hugovk** [python/blurb#33]
Following https://discuss.python.org/t/request-python-organisation-on-pypi/26545, let's ~~move~~ add this package as the second into the https://pypi.org/org/cpython/ organisation.

The main benefits are:

* Visibility: we'll be able to see our packages under https://pypi.org/org/cpython/

* Maintainability: we can share granular PyPI access to improve the bus factor

I should have the correct permissions to ~~move~~ add it and can do it tomorrow.

**Comment by larryhastings** [comment]
I remember when I first wrote blurb.  Mariatta informed me, you _must_ move it into my new `core_workflow` repository.  I was like, can't it just be on its own?  (I think I had it just in its own `blurb` repo.)  She said, no, absolutely not.  This is an important new initiative.  So I moved it into `core_workflow`.  The only other thing there was `blurb_it` as I recall.  Two months later Mariatta moved `blurb_it` out to its own repository.

Anyway, I don't have a strong opinion about it.
